src/app/shared/shared.module.ts

```diff
@@ -116,13 +112,17 @@ export function apiConfigProvider(configService: ConfigurationService, appStateS
     TableModule,
     TabViewModule,
     ToastModule,
-    TranslateModule
+    TranslateModule,
+    AngularRemoteComponentsModule
   ],
   //this is not elegant, for some reason the injection token from primeng does not work across federated module
   providers: [
+    {
+      provide: SLOT_SERVICE,
```

Comment:
Needs to be done in the remote module because otherwise we will not be able to get it running in standalone because the standalone needs a different slot service